documentation/domains/Domain.json

```diff
@@ -86,6 +86,22 @@
         }
       }
     },
+    "BaseServerPod": {
```

Comment:
This looks like an internal artifact that doesn't belong in the public schema?

Reply:
Renamed to `IntrospectorPod`.